app/src/main/res/values-cs/strings.xml

```diff
@@ -0,0 +1,441 @@
+  <string name="paymentdetails_fees_label">Poplatky</string>
+  <string name="paymentdetails_error_label">Chyba</string>
+  <string name="paymentdetails_closing_desc">Platba k uzavření kanálu #%1$s</string>
+  <string name="paymentdetails_no_description">Žádný popis</string>
+
+  <string name="paymentdetails_technicals_button">Zobrazit technická data</string>
+
+  <!-- //////////////// payment details: technicals data //////////////// -->
+
+  <string name="paymentdetails_technicals_title">Platební data</string>
+  <string name="paymentdetails_payment_hash_label">Platební Hash</string>
+  <string name="paymentdetails_payment_request_label">Platební Požadavek</string>
+  <string name="paymentdetails_preimage_label">Preimage</string>
+  <string name="paymentdetails_created_at_label">Vytvořeno v</string>
+  <string name="paymentdetails_expire_at_label">Vyprší v</string>
+  <string name="paymentdetails_completed_at_label">Dokončeno v</string>
+
+  <!-- //////////////// payment details: technicals data //////////////// -->
+
+  <string name="state_wait_confirmed">Spojovací transakce čeká na potvrzení</string>
+  <string name="state_closing">Kanál se uzavírá</string>
+  <string name="state_closed">Kanál byl uzavřen</string>
+  <string name="state_normal">Běží v pořádku</string>
+  <string name="state_sync">Synchronizuji s uzlem</string>
+  <string name="state_offline">Uzel je offline</string>
+  <string name="state_shutdown">Vypnuto</string>
+
+  <!-- //////////////// seed security //////////////// -->
+
+  <string name="seedsec_title">Bezpečnost</string>
+  <string name="seedsec_subtitle">Vysoce doporučujeme nastavit PIN k ochraně vaší peněženky. Pokud to vaše zařízení podporuje, můžete použít biometrické přihlášení.</string>
+  <string name="seedsec_pin_switch">Přihlášení pomocí PIN</string>
+  <string name="seedsec_biometrics_switch">Biometrické přihlášení</string>
+  <string name="seedsec_change_pin">Změnit PIN…</string>
+  <string name="seedsec_biometric_enrollment_error">Přístup k biometrii selhal. Prosím zkuste to znovu.</string>
+  <string name="seedsec_pindialog_title_unlock">Prosím zadejte váš stávající PIN</string>
+  <string name="seedsec_pindialog_title_set_new"><![CDATA[Prosím zadejte <b>váš nový PIN</b>]]></string>
+  <string name="seedsec_pindialog_title_confirm_new"><![CDATA[Prosím <b>potvrďte</b> váš nový PIN]]></string>
+
+  <string name="seedsec_pin_update_success">PIN úspěšně aktualizován!</string>
+  <string name="seedsec_biometric_enabled">Biometrické přihlášení aktivováno</string>
+  <string name="seedsec_biometric_disabled">Biometrické přihlášení deaktivováno</string>
+
+  <string name="seedsec_biometric_support_no_hw">Vaše zařízení nepodporuje biometrické přihlášení.</string>
+  <string name="seedsec_biometric_support_hw_unavailable">Biometrický hardware není dostupný. Prosím zkuste to znovu později.</string>
+  <string name="seedsec_biometric_support_none_enrolled">Prosím nejdříve povolte přístup k biometrii.</string>
```

Comment:
I don't think that "enroll" has same semantics as "allow access" here. I think it means "setup".